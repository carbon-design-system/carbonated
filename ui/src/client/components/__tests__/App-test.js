/**
 * Copyright IBM Corp. 2019, 2019
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

import React from 'react';
import ReactDOM from 'react-dom/server';
import { IntlProvider } from 'react-intl';

describe('App', () => {
  let App;
  let mountNode;

  beforeEach(() => {
    App = require('../App').default;
    mountNode = document.createElement('div');
    document.body.appendChild(mountNode);
  });

  afterEach(() => {
    document.body.removeChild(mountNode);
  });

  it('should render', () => {
    const tree = ReactDOM.renderToStaticMarkup(
      <IntlProvider locale="en">
        <App />
      </IntlProvider>,
      mountNode
    );
    expect(tree).toMatchSnapshot();
  });
});
