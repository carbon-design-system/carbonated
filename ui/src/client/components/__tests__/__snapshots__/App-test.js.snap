// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `"<div><span>Hello world</span></div>"`;
